doc/classes/Input.xml

```diff
@@ -295,6 +295,15 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="register_axis_as_button">
+			<return type="void" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="axis" type="int" enum="JoyAxis" />
+			<param index="2" name="deadzone" type="float" />
+			<description>
+				Treat axis with the id [param axis] (see [enum JoyAxis]) of the device with id [param device] as if it were not an axis, but two associated buttons, that get pressed when the axis reaches [param deadzone] on the negative and positive side respectively.
```

Comment:
```suggestion
				Treat axis with the id [param axis] (see [enum JoyAxis]) of the device with id [param device] as if it were not an axis, but two associated buttons, that get pressed when the axis reaches [param deadzone] on the negative and positive side respectively.
				For example, when [@GlobalScope.JOY_AXIS_LEFT_X] is registered as button, instead of [InputEventJoypadMotion] events it will send [InputEventJoypadButton] events with [member InputEventJoypadButton.button_index] being either [@GlobalScope.JOY_BUTTON_LEFT_X_NEG] or [@GlobalScope.JOY_BUTTON_LEFT_X_POS].
```

Reply:
Makes sense. Had to add `constant` to `@GlobalScope` constants.